DoxygenRule.cmake

```diff
@@ -88,6 +88,19 @@ if(NOT COMMON_ORGANIZATION_NAME)
   set(COMMON_ORGANIZATION_NAME Unknown)
 endif()
 
+# collect application help page
+get_property(__help GLOBAL PROPERTY ${PROJECT_NAME}_HELP)
```

Comment:
No, it would pick up stale files (see above for manual checks during release...)